java/arcs/android/crdt/ParcelableRawEntity.kt

```
@@ -67,10 +68,13 @@ data class ParcelableRawEntity(
                 collections[key] = set
             }
 
+            @Suppress("GoodTime") // use Instant
```

why not instant? is the comment a todo (in that case can it be rephrased as a todo?)?

---

iirc it's not cross platform, so i couldn't add it. this is an internal check, so we're just suppressing the error for now.

---

can you say that as a comment in the code ("we are not using instant because ....")